packages/optimizer/lib/transformers/MinifyHtml.js

```diff
@@ -131,7 +131,14 @@ class MinifyHtml {
   }
 
   async minifyAmpScript(child) {
+    if (!isDependencyInstalled('terser')) {
+      this.log_.warn(
```

Comment:
this is probably splitting hairs, but would `log_.error` be more appropriate? It seems this should be more severe than a warning

Reply:
Good point. Consistently printing an error now.